docs/general/ddl/data-types.rst

```diff
@@ -487,15 +487,7 @@ Time zone syntax as defined by `ISO 8601 time zone designators`_.
     +------------------+
     SELECT 1 row in set (... sec)
 
-    cr> select '13:59:59.999999 CET'::timetz;
-    +-----------------------+
-    | 13:59:59.999999+02:00 |
```

Comment:
Afaik this is wrong and should be fixed inside the implementation.
PostgreSQL:
```
postgres=# select '13:59:59.999999 CET'::timetz;
       timetz       
--------------------
 13:59:59.999999+01
(1 row)

postgres=# select '13:59:59.999999 CEST'::timetz;
       timetz       
--------------------
 13:59:59.999999+02
(1 row)
```

Reply:
The same result is now also returned by CrateDB:

```
crate=> select '13:59:59.999999 CET'::timetz;
 13:59:59.999999+01:00
-----------------------
 13:59:59.999999+01:00
(1 row)
```


The problem is that the output is in relation to UTC, and depending on the
daylight savings CET can be +2 or +1 hour.

Reply:
It is indeed a bit confusing. From https://time.is/nl/CET

> CET alternates between UTC+1 (standard time) and UTC+2 (when daylight saving time (DST) is observed). All countries in the CET time zone observe DST (UTC+2) from 02:00 am on the last Sunday of March until 03:00 am on the last Sunday of October.

> Warning: Many sources define CET as a constant UTC+1. In common usage however, CET usually refers to the time observed in most of Europe, be it standard time or daylight saving time.

Reply:
Hm I don't understand this as `CET` is always `+1` and never `+2` and opposite for `CEST` which always `+2`. What do I miss? ;)